CODEOWNERS

```diff
@@ -11,10 +11,10 @@ packages/website/  @BMillman19 @fragosti @fabioberger @steveklebanoff
 
 # Dev tools & setup
 .circleci/ @LogvinovLeon
-packages/abi-gen/ @LogvinovLeon
+packages/abi-gen/ @feuGeneA
 packages/base-contract/ @LogvinovLeon
```

Comment:
```suggestion
packages/base-contract/ @xianny 
```

Reply:
while we're at it.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1878/commits/53cee8d49e1b7d86f09c46b051fe2a5e50b30c4e